data/protein_quantification.csv

```diff
@@ -0,0 +1,27 @@
+,u87pi norm,h5m norm,plate
```

Comment:
I don't create it - collaborators at Dartmouth have (i use it in an R script for plotting only) `scripts/viz/viz_validation.R`